dear_petition/petition/tests/test_models.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+from .factories import BatchFactory, record_data
+
+from ..models import CIPRSRecord
+
+from ..utils import dt_obj_to_date
+
+pytestmark = pytest.mark.django_db
+
+
+def test_ciprs_record_create():
+    """Test that the create_record queryset method
+
+  This method should extract General, Case, and Defendant
+  details from the data field on the CIPRSRecord instance and store that
+  information in their fields
+  """
+    batch = BatchFactory()
+    data = record_data(1)
+    CIPRSRecord.objects.create_record(batch=batch, label=batch.label, data=data)
+    ciprs_record = CIPRSRecord.objects.get(data=data)
```

Comment:
Can you assert that the `batch` and `label` are what you would expect them to be as well?

Reply:
done